src/mon/MonMap.cc

```diff
@@ -823,6 +823,20 @@ seastar::future<> MonMap::build_monmap(const crimson::common::ConfigProxy& conf,
 
 seastar::future<> MonMap::build_initial(const crimson::common::ConfigProxy& conf, bool for_mkfs)
 {
+  // mon_host_override?
+  if (auto mon_host_override = conf.get_val<std::string>("mon_host_override");
```

Comment:
ack. this change was introduced in ed3782e60afa0da2f523fc6df7b593fe7a57646a, but the crimson's counterpart failed to catch up with the classic's implementation.